locales/en-US/experiments.ftl

```diff
@@ -143,7 +143,7 @@ snoozetabsContributors5Title = Firefox QA
 snoozetabsContributors6Title = Softvision QA
 snoozetabsContributors7Title = Softvision QA
 tabcenterDescription = What would it be like to move tabs from the top of the browser to the side? We wanted to find out!
-tabcenterIntroduction = <p>Take your tabs from in the way to tucked away. Tab Center moves your tabs to the side of your browser window so they're out of sight when you don't need them, and easy to expand and grab when you do.</p>
+tabcenterIntroductionWarning = <div class="warning"><strong>Firefox is changing fast. Tab Center may have bugs in Firefox 55 and will be disabled completely in Firefox 56+. </strong></div> <p>Take your tabs from in the way to tucked away. Tab Center moves your tabs to the side of your browser window so they're out of sight when you don't need them, and easy to expand and grab when you do.</p>
 tabcenterMeasurements0 = We collect usage data regarding how you interact with the Tab Center, such as how often you maximize or pin Tab Center.
```

Comment:
In the original PR, John had originally duplicated the `tabcenterMeasurements0`. Looks like it was removed, but GitHub thinks I commented on the one that stayed. This is resolved.